app.json

```diff
@@ -1,30 +1,31 @@
 {
-	"expo": {
```

Comment:
this file just got formatted by prettier. Since it wasn't a screen, we missed it when we first went through and used prettier on all of our screens.